torchio/data/subject.py

```diff
@@ -119,6 +119,12 @@ def get_composed_history(self):
         from ..transforms.augmentation.composition import Compose
         return Compose(self.history)
 
+    def get_undo_transform(self):
+        return self.get_composed_history().inverse()
+
+    def undo_transform(self):
+        return self.get_undo_transform()(self)
+
```

Comment:
that is a funny feature, we get from this PR ! well done !
I just wonder if we should call it apply_inverse_transform(). 
"undo" make you thing that will come back at the exact previous values ... but this will not be the case (depending on the transform)